source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -82,11 +84,13 @@ Settings
      - | **Type:** boolean
        |
        | **Description:**
-       | Whether to publish the changed document instead of the full change
-         stream document. When set to ``true``, the connector
-         automatically sets the ``change.stream.full.document`` property to
-         ``updateLookup`` to receive the updated documents.
-       |
+       | Whether to return only the ``fullDocument`` field from the full
+         change stream document. The ``fullDocument`` field contains the
+         most current version of the updated document.
```

Comment:
Suggestion:
Sorry if this is a new suggestion, but I was thinking that maybe linking to a server manual page on an explanation of "most current version" would be helpful since there are some details that might not be easy to summarize in a sentence.
Here's a page that talks about fullDocument for update operations.
https://www.mongodb.com/docs/manual/changeStreams/#lookup-full-document-for-update-operations

Also, change streams events can be inserts and deletes, so I think it could be useful to specify this is behavior for fullDocument applies to update events (perhaps tie it into the reason that the "true" setting overrides whatever setting exists in change.stream.full.document).